shared/constants/types/tracker2.tsx

```diff
@@ -74,8 +74,9 @@ export type NonUserDetails = {
   pictureUrl?: string
   formattedName?: string
   siteIcon: SiteIconSet
-  siteIconFull: SiteIconSet // full color icon,
-  siteIconWhite: SiteIconSet
+  siteIconDarkmode: SiteIconSet // full color icon,
```

Comment:
are you sure this comment is in the right place?

Reply:
oops